webhooks-extension/src/components/WebhookDisplayTable/WebhookDisplayTable.js

```diff
@@ -348,6 +362,7 @@ export class WebhookDisplayTable extends Component {
                 </fieldset>
               </Modal>
             </div>
+            {this.state.selectedWebhook !== null && <WebhookBranches webhook={this.state.selectedWebhook} close={this.closeBranches}/>}
```

Comment:
no need for the `!== null` unless it can have other falsey values
`{this.state.selectedWebhook && ...`